content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -2,264 +2,192 @@
+{{< /note >}}
 
-```yaml
-spec:
-  containers:
-    ...
-    resizePolicy:
-    - resourceName: cpu
-      restartPolicy: NotRequired
-    - resourceName: memory
-      restartPolicy: NotRequired
-    resources:
-      limits:
-        cpu: 700m
-        memory: 200Mi
-      requests:
-        cpu: 700m
-        memory: 200Mi
-...
-  containerStatuses:
-...
-    name: qos-demo-ctr-5
-    ready: true
-...
-    resources:
-      limits:
-        cpu: 700m
-        memory: 200Mi
-      requests:
-        cpu: 700m
-        memory: 200Mi
-    restartCount: 0
-    started: true
-...
-  qosClass: Guaranteed
+Check the pod status again after patching:
+
+```shell
+kubectl get pod resize-demo --output=yaml --namespace=qos-example
 ```
 
+You should see:
+* `spec.containers[0].resources` now shows `cpu: 800m`.
+* `status.containerStatuses[0].resources` also shows `cpu: 800m`, indicating the resize was successful on the node.
+* `status.containerStatuses[0].restartCount` remains `0`, because the CPU `resizePolicy` was `NotRequired`.
 
-## Updating the pod's resources
+## Example 2: Resizing Memory with Restart
 
-Let's say the CPU requirements have increased, and 0.8 CPU is now desired. This may
-be specified manually, or determined and programmatically applied by an entity such as
-[VerticalPodAutoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler#readme) (VPA).
+Now, let's resize the memory for the *same* pod. We'll increase it to `300Mi`. Since the memory `resizePolicy` is `RestartContainer`, we expect the container to restart.
 
-{{< note >}}
-While you can change a Pod's requests and limits to express new desired
-resources, you cannot change the QoS class in which the Pod was created.
-{{< /note >}}
+```shell
+kubectl patch pod resize-demo --subresource resize --patch \
+  '{"spec":{"containers":[{"name":"pause", "resources":{"requests":{"memory":"300Mi"}, "limits":{"memory":"300Mi"}}}]}}'
+```
 
-Now, patch the Pod's Container with CPU requests & limits both set to `800m`:
+Check the pod status shortly after patching:
 
 ```shell
-kubectl -n qos-example patch pod qos-demo-5 --subresource resize --patch '{"spec":{"containers":[{"name":"qos-demo-ctr-5", "resources":{"requests":{"cpu":"800m"}, "limits":{"cpu":"800m"}}}]}}'
+kubectl get pod resize-demo --output=yaml
 ```
 
-Query the Pod's detailed information after the Pod has been patched.
+You should now observe:
+* `spec.containers[0].resources` shows `memory: 300Mi`.
+* `status.containerStatuses[0].resources` also shows `memory: 300Mi`.
+* `status.containerStatuses[0].restartCount` has increased to `1` (or more, if restarts happened previously), indicating the container was restarted to apply the memory change.
+
+## Troubleshooting: Infeasible Resize Request
+
+Let's try requesting an unreasonable amount of CPU, like 1000 full cores (written as `"1000"` instead of `"1000m"` for millicores), which likely exceeds node capacity.
```

Comment:
```suggestion
Next, try requesting an unreasonable amount of CPU, such as 1000 full cores (written as `"1000"` instead of `"1000m"` for millicores), which likely exceeds node capacity.
```